basic/README.md

```diff
@@ -1,9 +1,9 @@
-Basic example
+Basic example using React Hooks
```

Comment:
Consider removing "using React Hooks" from all the readme's? If these examples are just being replaced with versions using hooks I don't think the extra messaging for that is necessary